### PR TITLE
Add CPU percentage and Cores metricset

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -219,11 +219,11 @@ update: python-env
 
 	# Update fields
 	echo "Update fields"
-	${PYTHON_ENV}/bin/python ${ES_BEATS}/libbeat/scripts/generate_fields_docs.py $(PWD) ${BEATNAME}
+	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/libbeat/scripts/generate_fields_docs.py $(PWD) ${BEATNAME}
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: @medcl This part was missing in your last PR

File: libbeat/scripts/Makefile [hunk 2]
```diff
@@ -219,11 +219,11 @@ update: python-env
```
Comment (medcl) [inline libbeat/scripts/Makefile]: hi @ruflin i think i call the “”python-env” before ”update：”
and it should have been actived already

Comment (ruflin) [inline libbeat/scripts/Makefile]: There are two steps: Creating the environment and using it. To use it in the Makefile here it must be part of the same call, that is why I have to add it to each line. Probably there is a better solution?

**Comment by ruflin** [comment]
https://github.com/elastic/beats/pull/1597 should be merged before this one.

**Comment by tsg** [comment]
See travis failure, looks like the `load` fields are not in fields.yml.

**Comment by ruflin** [comment]
Updated fields.yml and added system tests.

**Pull request merged**